src/proxyvisor.coffee

```diff
@@ -57,9 +60,13 @@ router.get '/v1/assets/:commit', (req, res) ->
 		return res.status(404).send('Not found') if !app
 		getAssetsPath(app.imageId)
 	.then (path) ->
-		getTarArchive(path)
-	.then (archivePath) ->
-		res.sendFile(archivePath)
+		dest = '/tmp/' + app.imageId + '.tar'
```

Comment:
that's interesting. It's clear that i made a wrong assumption around `imageId` property - will update